framework-core/src/main/resources/OSGI-INF/datatype-registry.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd rather migrating everything at once instead of having a different approaches. But yes that's something we should do if we want to tie ourselves to osgi.